apps/desktop/src/lib/posts/Post.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+	import { getContext } from '@gitbutler/shared/context';
+	import Interest from '@gitbutler/shared/redux/interest/Interest.svelte';
+	import { FeedService } from '@gitbutler/shared/redux/posts/service';
+	import { postSelectors, postRepliesSelectors } from '@gitbutler/shared/redux/posts/slice';
+	import { useStore } from '@gitbutler/shared/redux/utils';
+
+	type Props = {
+		postId: string;
+	};
+
+	const { postId }: Props = $props();
+
+	const feedService = getContext(FeedService);
+
+	const store = useStore();
+	const postWithRepliesInterest = $derived(feedService.getPostWithRepliesInterest(postId));
+	const post = $derived(postSelectors.selectById($store, postId));
+	const postReplies = $derived(postRepliesSelectors.selectById($store, postId));
```

Comment:
I'm wondering in how many instances of components we might end up reacting to `$store`, and if that could ever get problematic?

Reply:
Sounds like a good idea 👍

Reply:
How can we implement that here?